### PR TITLE
Rs installations

### PR DESCRIPTION
**Pull request by jbrezmorf**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -95,26 +95,28 @@ jobs:
...
       run: |
+        cp .gitmodules_https .gitmodules
         submodules/initialize.sh
     - name: Download prerequisites
       run: |
         wget -r -np -nd -nv -R "index.html*" -e robots=off -P build/win_x86 https://geomop.nti.tul.cz/prerequisites/
         cd build/win_x86
-        unzip gmsh-3.0.5-Windows64.zip
-        mv gmsh-3.0.5-Windows gmsh
+        unzip gmsh-4.9.1-Windows64.zip
```
Comment (jbrezmorf) [inline .github/workflows/build.yml:104]: - get just Python installer
- drop dependency on gmsh, we should use GSMH comming with bgem package

File: build/windows/win_x86.nsi [hunk 2]
```diff
@@ -133,7 +135,7 @@ Section "Runtime Environment" SecRuntime
   File /r /x *~ /x __pycache__ /x pylintrc /x *.pyc "${SRC_DIR}\gm_base"
 
   # Copy LICENSE, CHANGELOG, VERSION.
-  File "${GIT_DIR}\VERSION"
+  File "${GIT_DIR}\version.yml"
   File "${GIT_DIR}\CHANGELOG.md"
```
Comment (jbrezmorf) [inline build/windows/win_x86.nsi:139]: renamed to changes.md

File: build/windows/win_x86.nsi [hunk 3]
```diff
@@ -90,8 +91,8 @@ Function .onInit
                   Abort
       InstallPython:
```
Comment (jbrezmorf) [inline build/windows/win_x86.nsi:92]: Can we test for existing python and install only if not installed?
See: [discussion](https://find-error.com/questions/65721338/installer-cannot-find-python-in-path-environment-variable-directly-after-instal)
... possibly add installed python to the %path% 

Possibly use [PYNSIS](https://pypi.org/project/pynsist/) that comes with bundled python
but has different file format. So whole NSI file would need to be reproduced.

File: build/windows/win_x86.nsi [hunk 4]
```diff
@@ -144,62 +146,17 @@ Section "Runtime Environment" SecRuntime
...
   File /r "${BUILD_DIR}\gmsh"
 
   # Install intersections.
-  SetOutPath $INSTDIR
-  File /r "${GIT_DIR}\submodules\intersections"
+  #SetOutPath $INSTDIR
+  #File /r "${GIT_DIR}\submodules\intersections"
 
   # Install yaml_converter.
   SetOutPath $INSTDIR
```
Comment (jbrezmorf) [inline build/windows/win_x86.nsi:162]: Set flow123d installer to version 3.9.0.
Cleanup NSIS file. E.g: SecJobScheduler

File: .github/workflows/build.yml (hunk 1 above)
Comment (radeksrb) [reply .github/workflows/build.yml:106]: Hotovo.

File: build/windows/win_x86.nsi (hunk 2 above)
Comment (radeksrb) [reply build/windows/win_x86.nsi:139]: Hotovo

**Pull request merged**